plugins/modules/env.py

```diff
@@ -1028,6 +1030,7 @@ def main():
             cascade=dict(required=False, type='bool', default=False, aliases=['cascading']),
             force=dict(required=False, type='bool', default=False),
             wait=dict(required=False, type='bool', default=True),
+            datahub_start=dict(required=False, type='bool', default=True),
```

Comment:
Can you add the `datahub_start` argument to the `DOCUMENTATION` section at the top of this source file?

Reply:
Done